### PR TITLE
added module pool.js

### PR DESCRIPTION
**Pull request by fowziaelmi**

### CODE REVIEW

**Comment by fowziaelmi** [comment]
added pool.js for pg

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "koalla_holla",
   "version": "1.0.0",
-  "description": "",
+  "description": "This must stop: --------------- ![Nevar again](https://i.makeagif.com/media/8-22-2014/GO_DT4.gif)",
```
Comment (joshgulledge) [inline package.json:4]: what is this?

File: server/modules/pool.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const pg = require('pg');
+
+const config = {
+  database: 'koala_holla',
+  host: 'localhost',
+  port: 5432,
+  max: 10,
```
Comment (joshgulledge) [inline server/modules/pool.js:7]: what is max 10?

**Review by joshgulledge — COMMENTED** [review]
strange adds on here

**Pull request merged**